src/components/global/Header/Navigation.tsx

```diff
@@ -8,44 +9,37 @@ export default function Navigation() {
     const address = location.pathname;
     const docAddress = location.pathname.substring(0, 6);
     return (
-      <>
-        <div
-          id="hamburger"
-          className={styles.hamburger}
-          onClick={() => send("TOGGLE_HAMBURGER")}
-        />
-        <Link className={styles.logoLink} to="/" data-testid='mainLogo'>
-          <div className={styles.flexContainer}>
-            <div data-testid="home_logo" onClick={clickLogo} className={styles.logo} />
-            <h1 className={styles.branding}>API</h1>
-          </div>
-        </Link>
-        <HamburgerNavigation />
-        <nav id="navbar" className={`${styles.flexContainer} ${styles.navbar}`} data-id="navbar">
-          {[
-            ["/", "Home"],
-            ["/docs/", "Documentation"],
-            ["/api-explorer/", "API Explorer"],
-          ].map(([to, label], i) => {
-            return i != 2 ? (
-              <div data-id={to} key={to}>
-                <Link
-                  to={to}
-                  className={to === docAddress ? styles.selected : ""}
-                >
-                  {label}
-                </Link>
-              </div>
-            ) : (
-              <div data-id={to} key={to}>
-                <Link to={to} className={to === address ? styles.selected : ""}>
-                  {label}
-                </Link>
-              </div>
-            );
-          })}
-        </nav>
-      </>
+        <React.Fragment>
+            <div id='hamburger' className={styles.hamburger} onClick={() => send('TOGGLE_HAMBURGER')} />
+            <Link className={styles.logoLink} to='/' data-testid='mainLogo'>
+                <div className={styles.flexContainer}>
+                    <div data-testid='home_logo' onClick={clickLogo} className={styles.logo} />
+                    <h1 className={styles.branding}>API</h1>
+                </div>
+            </Link>
+            <HamburgerNavigation />
+            <nav id='navbar' className={`${styles.flexContainer} ${styles.navbar}`} data-id='navbar'>
+                {[
+                    ['/docs', 'Documentation'],
+                    ['/api-explorer/', 'API Explorer'],
+                    ['/app-registration/', 'Application Registration'],
+                ].map(([to, label], i) => {
+                    return i != 2 ? (
+                        <div data-id={to} key={to}>
+                            <Link to={to} className={to === docAddress ? styles.selected : ''}>
+                                {label}
+                            </Link>
+                        </div>
+                    ) : (
+                        <div data-id={to} key={to}>
+                            <Link to={to} className={to === address ? styles.selected : ''}>
+                                {label}
+                            </Link>
+                        </div>
+                    );
```

Comment:
These two conditions are more or less the same apart from using `docAddress` or `address`. Why the specific check for `i != 2`?